runtime/compiler/runtime/MetaData.cpp

```diff
@@ -1217,10 +1230,13 @@ static void populateInlineCalls(
          traceMsg(comp, "inlineIdx %d, callSiteCursor %p, inlinedCallSite->methodInfo = %p\n", i, callSiteCursor, inlinedCallSite->_methodInfo);
          }
 
-      if (!vm->isAOT_DEPRECATED_DO_NOT_USE()) // For AOT, we should only have returned resolved info about a method if the method came from same class loaders.
+      if (!vm->isAOT_DEPRECATED_DO_NOT_USE()
+#if defined(JITSERVER_SUPPORT)
+         && TR::comp()->getPersistentInfo()->getRemoteCompilationMode() != JITServer::SERVER
```

Comment:
Since we have the comp object, a better test could be comp->isOutProcessCompilation()
comp is passed as parameter, don't need to use TR::comp()